### PR TITLE
Documentation: Fist batch of concept updates #566

### PR DESCRIPTION
**Pull request by bari12**
Documentation: Fist batch of concept updates #566

### CODE REVIEW

File: doc/source/overview_File_Dataset_Container.rst [hunk 1]
```diff
@@ -2,24 +2,22 @@
 Files, Datasets and Containers
 ------------------------------
 
-ATLAS has a large amount of data, which is physically stored in files. For the data management system these files are the smallest operational unit of data (so sub-file operations are not possible). Physicists need to be able to identify and operate on any arbitrary set of files.
+As data is physically stored in files, files are also the smalles operational unit of data in Rucio. Sub-file operations are currently not possible. Rucio enables users to identify and access on any arbitrary set of files.
```
Comment (vingar) [inline doc/source/overview_File_Dataset_Container.rst:5]: tmp_dids should maybe described in some incubation work pages ? and link to it ?

File: doc/source/overview_File_Dataset_Container.rst [hunk 2]
```diff
@@ -2,24 +2,22 @@
...
+The scope string partitions the name space into several sub namespaces. The primary use case for this is to easily separate centrally created data from individual user data.
 
-By default accounts will have read access to all scopes and write access only to their own scope. Privileged accounts will have write access to multiple scopes, e.g., production might use scopes such as mc11, data12_8TeV, tmp.prod.#
+By default accounts have read access to all scopes and write access only to their own scope. Privileged accounts have write access to multiple scopes, e.g., the Workload Management System is allowed to write into official data scopes.#
 
-Files, datasets and containers are uniquely identified over all time. This implies that an identifier, once used, can never be reused to refer to anything else at all, not even if the data it referred to has been deleted from the system.
+Files, datasets and containers are uniquely identified over all time. This implies that a data identifier, once used, can never be reused to refer to anything else at all, not even if the data it referred to has been deleted from the system.
```
Comment (vingar) [inline doc/source/overview_File_Dataset_Container.rst:21]: do we have concrete examples in this section ?

File: doc/source/overview_File_Dataset_Container.rst [hunk 3]
```diff
@@ -38,9 +36,7 @@ A file is LOST if there are no known replicas of the file in Rucio, while at the
 
 * ``suppressed``: True/False
 
-This is a user settable flag. It indicates that the owner of the scope no longer needs the name to be present in the scope. Files that are suppressed (by default) do not show up in search and list operations on the scope. The setting of this flag is subject to conditions, e.g., one can not suppress a file while at the same time requesting it to be replicated somewhere.
-
-Note however that this flag will be ignored when explicitly listing contents of datasets/containers.
+This is a user settable flag. It indicates that the owner of the scope no longer needs the name to be present in the scope. Files that are suppressed (by default) do not show up in search and list operations on the scope. Note however that this flag will be ignored when explicitly listing contents of datasets/containers.
```
Comment (vingar) [inline doc/source/overview_File_Dataset_Container.rst:40]: suppressed is not really used in Rucio but more a external/worklow field no ?

File: doc/source/overview_Rucio_account.rst [hunk 4]
```diff
@@ -3,25 +3,18 @@ Rucio account
 -------------
 
 A Rucio account is the unit of assigning privileges in Rucio. It can
-represent individual users (like lgoossen, graemes, vgaronne, ...), a
-group of users (like bphys, higgs, susy, ...) or an organised
-production activity for the whole ATLAS collaboration (prod,
-tzero, ...). A Rucio account is identified by a string.
+represent individual users (such as user1, user2, user3, ...), a
+group of users (such as group1, group2, group3, ...) or a centralized production
+activity (such as datagen, wfms, ...). A Rucio account is identified by a string.
```
Comment (vingar) [inline doc/source/overview_Rucio_account.rst]: wfms = the service account related to a workflow management system interfaced with Rucio

File: doc/source/overview_File_Dataset_Container.rst (hunk 1 above)
Comment (bari12) [reply doc/source/overview_File_Dataset_Container.rst:5]: Maybe at some later stage, I don't think it is a major concept which we have to feature that prominently. I'll put it to the todo list.

File: doc/source/overview_File_Dataset_Container.rst (hunk 2 above)
Comment (bari12) [reply doc/source/overview_File_Dataset_Container.rst:21]: No, we did have some dataset/container hierarchy image at some point, but I can't find it anymore in git history...

File: doc/source/overview_File_Dataset_Container.rst (hunk 3 above)
Comment (bari12) [reply doc/source/overview_File_Dataset_Container.rst:40]: Yep. To be honest I wanted to remove it already from the page and just left it in as it was in the previous version. But I would be happy to remove it. It is not really significant.

File: doc/source/overview_Rucio_account.rst (hunk 4 above)
Comment (bari12) [reply doc/source/overview_Rucio_account.rst]: Will add.

File: doc/source/concepts.rst [hunk 5]
```diff
@@ -1,15 +1,15 @@
 Concepts and terminology
 ========================
 
-The following concepts define how data is organised, accessed, and catalogued by Rucio.
+The following concepts define how data is organised, accessed, and catalogued by Rucio. Some concepts, such as accounts, data identifiers, or replication rules are fundamental for all types of users, while other concepts, such as notifications or subscriptions, are aimed at more advanced users.
```
Comment (mlassnig) [inline doc/source/concepts.rst]: ... are suitable for more advanced use cases.

**Comment by bari12** [comment]
Ping, somebody review please, so I can merge this 😄

**Pull request merged**